models/DataObject/ClassDefinition/Data/Wysiwyg.php

```diff
@@ -105,6 +117,10 @@ public function getDataForResource(mixed $data, DataObject\Concrete $object = nu
      */
     public function getDataFromResource(mixed $data, DataObject\Concrete $object = null, array $params = []): ?string
     {
+        $helper = self::getWysiwygSanitizer();
+        if (is_string($data)) {
+            $data = $helper->sanitize(html_entity_decode($data));
+        }
```

Comment:
```suggestion
        if (is_string($data)) {
            $data = self::getWysiwygSanitizer()->sanitize(html_entity_decode($data));
        }
```

Or at least only get it when necessary:

```suggestion
        if (is_string($data)) {
            $helper = self::getWysiwygSanitizer();
            $data = $helper->sanitize(html_entity_decode($data));
        }
```

Reply:
@jdreesen thank you for these suggestions! Should be adapted now.